HOWTO.rst

```diff
@@ -3826,6 +3826,13 @@ Steady state
 	collection for checking the steady state job termination criterion. The
 	default is 0.  When the unit is omitted, the value is interpreted in seconds.
 
+.. option:: steadystate_check_interval=time, ss_interval=time
+
+	The values suring the rolling window will be collected with a period
```

Comment:
during instead of suring. Also I think it would be useful to clarify that if `ss_dur`=60s and the check interval is 6s only 10 measurements will be collected.